src/cli/msig/symbolMigrate.ts

```diff
@@ -215,3 +215,50 @@ export async function migrateRebrandApps(options: StandardProposalOptions) {
 
     if (!options.dryRun && options.autoExecute) await executeProposal(options.proposer, proposalName, proposalHash);
 }
+
+export async function migrateRebrandAppsV2(options: StandardProposalOptions) {
+    console.log('### Migrating rebranding');
+
+    const apps = await TonomyContract.Instance.getApps();
+    const actions = await apps
+        .filter((app) => app.origin.includes('tonomy.io'))
```

Comment:
instead of creating a new migration, update the previous migration.

update the filter so that instead of checking the origin it checks if any of these are true:
- app name contains pangea or leos
- app description contains pangea or leos
- app origin contains pangea
- app name contains sales platform

then make some conditional logic to determine how the app should update, which has to work on mainnet and testnet (your current code only works for testnet)